pkg/controllers/ipam/ipam.go

```diff
@@ -358,6 +441,11 @@ func (i *Manager) AssignIP(ctx context.Context, allocate *svcRecord, svc *v1.Ser
 	}
 	svc.Labels[constant.OpenELBEIPAnnotationKeyV1Alpha2] = allocate.Eip
 
+	if svc.Annotations == nil {
+		svc.Annotations = make(map[string]string)
+	}
+	svc.Annotations[constant.OpenELBProtocolAnnotationKey] = eip.GetProtocol()
+
```

Comment:
OpenELB does not support ip assignment for the specified protocol. At the same time, in order to reduce user costs, users can no longer specify `protocol.openelb.kubesphere.io/v1alpha1`.
The `controller` removes validation about the protocol. Therefore, even if the user specifies the protocol, it will have no effect on the controller;
The `speaker` behaves differently depending on the eip used, especially vip mode (we need to update configmap/openelb-vip-configmap). The required protocol can be obtained from eip. If eip is deleted or cannot be found, the speaker will delete the announcement of the route, and at this time there will be related records in the memory.
So I don't think there is any need to backfill the protocol information here. how do you think? @chaunceyjiang